src/layers/core/scatterplot-layer/scatterplot-layer.js

```diff
@@ -35,15 +36,21 @@ const defaultProps = {
   radiusMinPixels: 0, //  min point radius in pixels
   radiusMaxPixels: Number.MAX_SAFE_INTEGER, // max point radius in pixels
   outline: false,
-  strokeWidth: 1
+  strokeWidth: 1,
+  fp64: false
 };
 
 export default class ScatterplotLayer extends Layer {
   getShaders(id) {
-    return {
-      vs: readFileSync(join(__dirname, './scatterplot-layer-vertex.glsl'), 'utf8'),
-      fs: readFileSync(join(__dirname, './scatterplot-layer-fragment.glsl'), 'utf8')
-    };
+
+    const shaders = this.props.fp64 ? {
+      vs: readFileSync(join(__dirname, './scatterplot-layer-64-vertex.glsl'), 'utf8'),
+      fs: readFileSync(join(__dirname, './scatterplot-layer-fragment.glsl'), 'utf8'),
+      modules: ['lighting', 'fp64', 'project64']} : {
```

Comment:
Nit: I'd put a new line before the `} : {` to more clearly show the two objects.